src/components/Layouts/ResetPasswordContainer.tsx

```diff
@@ -0,0 +1,32 @@
+import backgroundImage from '../../assets/register-login-background.svg';
+import logo from '../../assets/otherLogo.svg';
+import { ReactNode } from 'react';
```

Comment:
same case here, import your asset from main file, image.ts

Reply:
thanks, let me change it